lib/src/config/custom_values.rs

```diff
@@ -7,24 +7,24 @@ use config::{Result, Config, Value, ConfigError};
 use http::Key;
 
 #[derive(Clone)]
-pub enum SessionKey {
+pub enum SecretKey {
     Generated(Key),
     Provided(Key)
 }
 
-impl SessionKey {
+impl SecretKey {
     #[inline(always)]
     pub fn kind(&self) -> &'static str {
         match *self {
-            SessionKey::Generated(_) => "generated",
-            SessionKey::Provided(_) => "provided",
+            secretKey::Generated(_) => "generated",
```

Comment:
Looks like `secretKey` should be `SecretKey`. Just poking around and spotted this.